src/server/auth/authRoutes.js

```diff
@@ -25,7 +26,7 @@ router.post(
         return res.status(400).send()
       }
 
-      const user = await db.User.build({ email, password })
+      const user = await db.User.create({ email, password })
       await user.save()
```

Comment:
If you're using the model's `create` the transaction is complete, `save` is not required. `build` creates a transaction, hence `save`.